### ISSUE TITLE
Some build.xml error when running via DITA-OT 2.2.3

### ISSUE THREAD

**Issue by ToshihikoMakita** [oasis-open/dita-rng-converter#1]
line 29 or near

``` xml
<property name="doctypesDir" location="${dita-ot-dir}/plugins/org.oasis-open.dita.dita13.doctypes/doctypes"/>
```

should be:

``` xml
<property name="doctypesDir" location="${dita-ot-dir}/plugins/org.oasis-open.dita.v1_3"/>
```

line 147 or near

``` xml
      <include name="resolver.jar"/>
```

resolver.jar is not included in DITA-OT 2.2.3. It contains xml-resolver-1.2.jar.

**Comment by drmacro** [comment]
I have yet to test the code with the 2.x OT. But this should be easy to fix. You can fix it locally by changing "resolver.jar"  to "xml-resolver-1.2.jar" in that include statement.

**Comment by drmacro** [comment]
I've resolved this by fixing the name of the 1.3 grammar plugin, by moving the Saxon9.7 HE jar into the project, and updating the classpath to handle any names for the required jars.

The OT through 2.3 ships with Saxon 9.1 and this transform requires 9.6+ so easiest solution is to include the jars in the project.
